package.json

```diff
@@ -43,6 +44,8 @@
     "sane": "^4.0.0",
     "tmp": "0.0.33",
     "tree-sync": "^1.2.2",
+    "ts-node": "^8.0.2",
+    "typescript": "^3.3.3",
```

Comment:
you'll certainly want to make this `~3.3.3`, since typescript does not follow semver, and regularly includes breaking changes in "middle-number" releases

Reply:
Dependency removed